Gruntfile.js

```diff
@@ -341,15 +341,17 @@ module.exports = function(grunt) {
           },
           {
             cwd: 'build/docs',
-            src: ['**', '!ptore2e/**'],
+            src: ['**', '!ptore2e/**', '!index*.html'],
             dest: 'deploy/docs/',
             expand: true
           },
           {
-            src: ['build/docs/index-production.html'],
-            dest: docsScriptFolder + '/functions/content',
-            expand: true,
-            flatten: true
+            src: 'build/docs/index-production.html',
+            dest: 'deploy/docs/index.html'
```

Comment:
We support escaped_fragment (Ajax crawling), and we serve the fragments from the functions folder, because functions don't have access to the hosted files